### DISCUSSION TITLE
Odd behaviour with multi container list.

### DISCUSSION THREAD

**Discussion by jgalminas** [clauderic/dnd-kit#630] [Q&A] [answered]
![ezgif com-gif-maker](https://user-images.githubusercontent.com/63853949/154856022-9c170d6d-1768-4b5e-85a0-1c35f3572b3e.gif)

Hi, I was wondering if anyone has encountered a similar issue to this. A neighboring container task with the same index flickers when swapping containers over.

**Comment by jgalminas** [comment]
Can provide code or more information if needed, any insight would be appreciated. Thanks!

**Comment by jgalminas** [comment, marked as answer]
I managed to fix this by adding  `        measuring={{
          droppable: {
            strategy: MeasuringStrategy.Always,
          },
        }}` to the DndContext component.

**Reply by sebastianservat** [reply]
Thanks for sharing @jgalminas!!! I had a similar flickering issue, found your fix and it worked!

Curious, how did you find the answer?

**Reply by jgalminas** [reply]
That's great! To be honest, it was just a lucky guess. I thought it might have something to do with collision and then saw that the examples contained that bit of code, tried it and it worked :)
